wiki/Beatmap/Converts/en.md

```diff
@@ -0,0 +1,48 @@
+---
+tags:
+  - convert beatmaps
+  - converted beatmaps
+---
+
+# Converts
+
+![The \"Show converted\" toggle in osu!(lazer)](img/show-converted-button.png "The \"Show converted\" toggle in osu!(lazer)")
+
+In osu! gameplay, **converts** (or *converted beatmaps*) are [beatmaps](/wiki/Beatmap) of a specific [game mode](/wiki/Game_mode) that have been "converted" by the osu! client in-game in order to be playable in another game mode they were not originally created for. All converts are automatically generated based on certain parameters present in the base beatmap.
```

Comment:
several things to unpack here:
- "gameplay" has nothing to do with it, and come to think of it, "In osu!" does too
- since only osu! game mode beatmaps can be converts, why not say that instead of "a specific game mode"?
- while the first sentence is factually correct, it gives an impression that these difficulties are permanent (as in, they coexist with regular ones), which isn't the case -- at least to me, they're automatically generated _versions_ of the original difficulties. how about you put focus on the second sentence instead?

Reply:
deleted the "in osu! gameplay" part

i wrote this section as such because i vaguely remember peppy saying that osu!(lazer) is planning to support conversion between modes e.g. taiko --> mania going forward

will try to ask peppy whether this is true or is this is just my memory hazing up tho...